can/io/player.py

```diff
@@ -88,6 +92,23 @@ def __new__(  # type: ignore
                 f'No read support for this unknown log format "{suffix}"'
             ) from None
 
+    @staticmethod
+    def unzip(zipfile: "can.typechecking.StringPathLike"):
```

Comment:
I'd suggest to rename this to `decompress()`, since it's not about `.zip` files and we could easily extend this to other compression types if needed.

Reply:
```suggestion
    def unzip(zipfile: "can.typechecking.StringPathLike") -> Tuple[str, typing.IO[typing.Any]]:
```

Reply:
will do!